lib/zold/node/farmers.rb

```diff
@@ -112,5 +115,31 @@ def kill(pid, start)
         @log.debug("No need to kill process ##{pid} since it's dead already: #{e.message}")
       end
     end
+
+    # In a child process using fork
+    class Fork
+      def initialize(log: Log::NULL)
+        @log = log
+      end
+
+      def up(score)
+        start = Time.now
+        read, write = IO.pipe
+        Process.fork do
+          score = score.next
+          write.puts "#{score.to_s}|#{Process.pid}"
+        end
+        Process.wait
+        write.close
+        output = read.read
+        buffer = output.split('|')[0]
+        proc_pid = output.split('|')[1]
```

Comment:
@emilianodellacasa how many more `pid`-s you have here? https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
@yegor256 Done